Modules/xxlimited.c

```diff
@@ -19,6 +19,8 @@
           def __init__(self):
               # In the C class, "_x_attr" is not accessible from Python code
               self._x_attr = {}
+              self._x_buffer = bytesarray(10)
+              self._x_exports = 0
```

Comment:
IMO, it's better to explain this in English than in Python.
The name `_x_buffer` isn't used elsewhere, so there's not much point in introducing it.
```suggestion
```